gravitee-apim-console-webui/src/management/api/creation/v4/models/ApiCreationStepService.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+
+import { ApiCreationPayload } from './ApiCreationPayload';
+import { ApiCreationStep, ApiCreationStepperService } from './ApiCreationStepperService';
+
+/**
+ * This service is injected in each step component to provide a way to access the payload and to navigate between steps.
+ */
+@Injectable()
+export class ApiCreationStepService {
```

Comment:
The file naming does not match usual conventions ApiCreationStepService.ts should be api-creation-step.service.ts.
Same for ApiCreationPayload and ApiCreationStepperService

Reply:
yes done,  now is not juste a class but a service